packages/examples/src/geometry-domain/textbook_problems/c11p25.sub

```diff
@@ -1,11 +1,12 @@
 Point A, B, C, D
 Circle c
 Segment AB := Chord(c, A, B)
-Segment CD := Chord(c, C, D)
+Segment DC := Chord(c, D, C)
```

Comment:
I'm curious, is there a particular reason you flipped this?

Reply:
Tricky case this one: `MkQuadrilateral` don't enforce vertex ordering and I haven't found a great way to express this as a constraint. The temporary solution is to use `Parallel`, which implicitly favors one direction and thereby makes the vertices ordered correctly